x-pack/plugins/reporting/server/export_types/csv_searchsource/generate_csv/generate_csv.ts

```diff
@@ -60,6 +60,7 @@ function isPlainStringArray(
 }
 
 export class CsvGenerator {
+  private _columns: string[] | null = null;
```

Comment:
I see that pattern in this class but isn't that easier (and cleaner) to make it optional and skip the initialization? It's a private field, shouldn't affect anything outside but will keep the instance clean.
```suggestion
  private _columns?: string[];
```